### PR TITLE
Do not save loss and check for loss in Pytorch Data Parallel

### PR DESCRIPTION
**Pull request by NihalHarish**
### Description of changes:
- Dataparallel test checks if smdebug supports pytorch data parallel.
- There is currently a [bug](https://github.com/awslabs/sagemaker-debugger/issues/205) in saving losses 
- Removing logic that saves loss and checks for it in this test, since it is not required to check the sanity of data parallel functionality 

#### Style and formatting:

I have run `pre-commit install` to ensure that auto-formatting happens with every commit.

#### Issue number, if available

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by NihalHarish** [comment]
See #217

File: tests/pytorch/utils.py [hunk 1]
```diff
@@ -48,7 +48,6 @@ def train(model, hook, device, optimizer, num_steps=500, set_modes=False):
         optimizer.zero_grad()
         output = model(Variable(data, requires_grad=True))
         loss = F.nll_loss(output, target)
-        hook.record_tensor_value("nll_loss", tensor_value=loss)
```
Comment (vandanavk) [inline tests/pytorch/utils.py:51]: if this line is removed, will it still save loss for vanilla PT ?

Comment (NihalHarish) [reply tests/pytorch/utils.py]: Reverting this.
